adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -49,501 +37,55 @@ + (nonnull GADMAdapterVungleRouter *)sharedInstance {
+                      [del initialized:error == nil error:error];
+                    }
+                    [self->_delegates removeAllObjects];
+                  }
+                }];
 }
 
-- (nullable NSError *)renderBannerAdInView:(nonnull UIView *)bannerView
-                                  delegate:(nonnull id<GADMAdapterVungleDelegate>)delegate
-                                    extras:(nullable VungleAdNetworkExtras *)extras
-                            forPlacementID:(nonnull NSString *)placementID {
-  NSMutableDictionary *options = nil;
-  if (extras) {
-    options = [[NSMutableDictionary alloc] init];
-    if (extras.muteIsSet) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(options, VunglePlayAdOptionKeyStartMuted,
-                                                        @(extras.muted));
-    }
-    if (extras.userId) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(options, VunglePlayAdOptionKeyUser,
-                                                        extras.userId);
-    }
-    if (extras.flexViewAutoDismissSeconds) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(
-          options, VunglePlayAdOptionKeyFlexViewAutoDismissSeconds,
-          @(extras.flexViewAutoDismissSeconds));
-    }
-  }
-
-  NSError *bannerError = nil;
-
-  BOOL success = [VungleSDK.sharedSDK addAdViewToView:bannerView
-                                          withOptions:options
-                                          placementID:placementID
-                                                error:&bannerError];
-
-  if (success) {
-    // For a refresh banner delegate, if the Banner view is constructed successfully,
-    // it will replace the old banner delegate.
-    if (delegate.isRefreshedForBannerAd) {
-      [self replaceOldBannerDelegateWithDelegate:delegate];
-    }
-  }
-  return bannerError;
-}
-
-- (void)completeBannerAdViewForPlacementID:(nonnull id<GADMAdapterVungleDelegate>)delegate {
-  @synchronized(self) {
-    if (!delegate || ![delegate respondsToSelector:@selector(bannerAdSize)]) {
-      return;
-    }
-
-    if (delegate.bannerState == BannerRouterDelegateStatePlaying ||
-        delegate.bannerState == BannerRouterDelegateStateWillPlay) {
-      NSLog(@"Vungle: Triggering an ad completion call for %@", delegate.desiredPlacement);
-      [VungleSDK.sharedSDK finishDisplayingAd:delegate.desiredPlacement];
-    }
-  }
-}
-
-- (void)initialized:(BOOL)isSuccess error:(nullable NSError *)error {
-  _isInitializing = NO;
-  NSMapTable<NSString *, id<GADMAdapterVungleDelegate>> *delegates = [_initializingDelegates copy];
-  for (NSString *key in delegates) {
-    id<GADMAdapterVungleDelegate> delegate = [delegates objectForKey:key];
-    [delegate initialized:isSuccess error:error];
-  }
-
-  [_initializingDelegates removeAllObjects];
-  _prioritizedPlacementID = nil;
+- (BOOL)isSDKInitialized {
+  return [VungleAds isInitialized];
 }
 
 - (NSString *)getSuperToken {
```

Comment:
Similar to this, I think we can just directly call `[VungleAds getBiddingToken]` instead of needing this util method.

Reply:
Updated.